cmd/resources/gen_resources.go

```diff
@@ -0,0 +1,138 @@
+package resources
+
+import (
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/getkin/kin-openapi/openapi3"
+)
+
+type TemplateData struct {
+	Resources map[string]*ResourceData
+}
+
+type ResourceData struct {
+	Name        string
+	Description string
+	Operations  map[string]*OperationData
```

Comment:
it's probably not strictly necessary to have these as a `map[string]*OperationData` vs `[]*OperationData` but it was helpful for pulling out a specific op type for testing.